srcs/parse/map/parse_map.c

```
@@ -0,0 +1,417 @@
+	i = 0;
+	while (i < map_lines)
+	{
+		normalized_map[i] = calloc(max_len + 1, sizeof(char));
+		if (!normalized_map[i])
+		{
+			while (i > 0)
+				free(normalized_map[--i]);
+			free(normalized_map);
+			return (-1);
+		}
+		while (j < max_len)
```

Cppcheck (2.18.0)

[warning] Uninitialized variable

(uninitvar)